### PR TITLE
Ignore more backend errors

### PR DESCRIPTION
**Pull request by bmwiedemann**
Ignore more backend errors - similar to #8199

except now the message is

    <status code="400"> <summary>home:regataos/gcc12: conflict in file gcc12.spec</summary> </status>

### CODE REVIEW

File: src/api/config/initializers/airbrake.rb [hunk 1]
```diff
@@ -95,6 +95,10 @@ def ignore_by_backend_400_message?(message)
   messages_to_ignore.each do |ignored_error_message|
     return true if message.include?(ignored_error_message)
   end
+  messages_to_ignore_re = ['<summary>[^<>]*: conflict in file [^<>]*</summary>'].freeze
```
Comment (danidoni) [inline src/api/config/initializers/airbrake.rb:98]: What about merging the previous `messages_to_ignore` list and this one? I have the impression that maybe this regexp would cover some of the previous errors also...

Comment (hennevogel) [reply src/api/config/initializers/airbrake.rb:98]: It's already included above: `message.include?('<summary>conflict in file')` right?

**Review by hennevogel — CHANGES_REQUESTED** [review]

File: src/api/config/initializers/airbrake.rb (hunk 1 above)
Comment (bmwiedemann) [reply src/api/config/initializers/airbrake.rb]: I omitted the `: ` so that now the Regexp matches both.

Comment (hennevogel) [reply src/api/config/initializers/airbrake.rb]: I don't get it, the message changed or what happened?

Comment (bmwiedemann) [reply src/api/config/initializers/airbrake.rb]: yes the part with `prj/pkg: ` got added.

Comment (danidoni) [reply src/api/config/initializers/airbrake.rb]: See [here](https://github.com/openSUSE/open-build-service/pull/14444/files#diff-567a729df5a343260e81707d6c050fb07cfd16878914491a892f24e866f3161aR14-R15) to get more details

**Review by hennevogel — CHANGES_REQUESTED** [review]

**Review by hennevogel — APPROVED** [review]

**Pull request merged**